.docksal/commands/init

```diff
@@ -96,6 +96,9 @@ fin reset -f
 echo "Waiting 10s for MySQL to initialize.";
 sleep 10
 
+# Symlink Drush
+fin exec '([ ! -f /usr/local/bin/drush ] || false) && sudo ln -sF /var/www/vendor/bin/drush /usr/local/bin/drush'
```

Comment:
@sean-e-dietrich I have the following on PCG.  What should it be, `true` or `false`?

`fin exec '([ -f /usr/local/bin/drush ] || true) && sudo ln -sf /var/www/vendor/bin/drush /usr/local/bin/drush'`